lib/monaco-adapter.js

```diff
@@ -133,49 +160,7 @@ firepad.MonacoAdapter = (function () {
         this.monaco.executeEdits("my-source", [ops]);
       }
     }
-    return this.grabDocumentState();
-  };
-
-  // Index in monaco starts from 1
-  MonacoAdapter.prototype.posFromIndex = function (index, content) {
-    var line, row, _i, _len, _ref;
-    if (content == null) {
-      _ref = this.lastDocLines;
-    }
-    else {
-      _ref = content;
-    }
-    for (row = _i = 0, _len = _ref.length; _i < _len; row = ++_i) {
-      line = _ref[row];
-      if (index <= line.length) {
-        break;
-      }
-      index -= line.length + this.monacoModel.getEOL().length;
-    }
-    return {
-      row: row + 1,
-      column: index + 1
-    };
-  };
-
-  // Index in monaco start from 1
-  MonacoAdapter.prototype.indexFromPos = function (range, upto, lines) {
-    var index;
-    if (lines == null) {
-      lines = this.lastDocLines;
-    }
-    index = 0;
-    if (upto === 'start') {
-      for (row = 1; row < range.startLineNumber; row++) {
-        index += lines[row - 1].length + this.monacoModel.getEOL().length;
-      }
-      return index + range.startColumn - 1;
-    } else {
-      for (row = 1; row < range.endLineNumber; row++) {
-        index += lines[row - 1].length + this.monacoModel.getEOL().length;
-      }
-      return index + range.endColumn - 1;
-    }
+    this.grabDocumentState();
```

Comment:
Was the return value not being used anywhere?

Reply:
No, it was specific for Ace editor, as Coffescript to my knowledge expects a return statement. This method updates internal states only, Editor client does not expect any return value.